code/game/atoms.dm

```diff
@@ -207,7 +210,7 @@
 	// *****RM
 	//user << "[name]: Dn:[density] dir:[dir] cont:[contents] icon:[icon] is:[icon_state] loc:[loc]"
 
-	if(reagents && is_open_container()) //is_open_container() isn't really the right proc for this, but w/e
+	if(reagents && is_open_container() || reagents && is_semi_open_container()) //is_open_container() isn't really the right proc for this, but w/e

```

Comment:
You MIGHT need to do `if(reagents && (is_open_container() || is_semi_open_container()))`. I do not remember the order of operations off the top of my head. Test both cases.